components/org.wso2.carbon.status.dashboard.web/src/siddhi-app/AppSpecific.jsx

```diff
@@ -164,34 +164,58 @@ export default class WorkerSpecific extends React.Component {
 
     componentWillMount() {
         let that = this;
-        AuthenticationAPI.isUserAuthorized('metrics.manager',AuthManager.getUser().token)
+        AuthenticationAPI.isUserAuthorized('metrics.manager',AuthManager.getUser().SDID)
             .then((response) => {
                 that.setState({
                     hasManagerPermission: response.data
                 });
             }).catch((error) => {
-            let re = /The session with id '((?:\\.|[^'])*)'|"((?:\\.|[^"])*)" is not valid./;
-            let found = error.response.data.match(re);
-            if (found != null) {
+            let message;
+            if(error.response != null){
```

Comment:
space before {.. check other places in this file

Reply:
Fixed in b4a2bcf444d25031775a679bad76798d70ce7d5b